api/pyproject.toml

```diff
@@ -34,4 +34,4 @@ known_third_party=['_pytest','apiclient','app_analytics','axes','chargebee','cor
 skip = ['migrations','.venv','.direnv']
 
 [tool.pytest.ini_options]
-addopts = ['--ds=app.settings.test', '-vvvv', '-p', 'no:warnings', '-n', 'auto']
+addopts = ['--ds=app.settings.test', '-vvvv', '-p', 'no:warnings' ]
```

Comment:
Heh, I think this was a mistake? (but one that I've probably made too because Pycharm debugging doesn't seem to work with multiple processes).

Reply:
We can move `xdist` options over to the `Makefile`. We'll have to do so for coverage too as `pytest-cov` disrupts the debugger, might as well move `xdist` there too.

Reply:
@khvn26 do you want to create a separate PR for this?

Reply:
> 



> Heh, I think this was a mistake? (but one that I've probably made too because Pycharm debugging doesn't seem to work with multiple processes).

Yep, it was a mistake - but yeah we should do it